apps/meteor/client/views/marketplace/AppMenu.js

```diff
@@ -334,20 +341,19 @@ function AppMenu({ app, isAppDetailsPage, ...props }) {
 						action: handleDisable,
 					},
 				}),
-			...(app.installed &&
-				isAdminUser &&
-				!isAppEnabled && {
-					enable: {
-						label: (
-							<>
-								<Icon name='check' size='x16' marginInlineEnd='x4' />
-								{t('Enable')}
-							</>
-						),
-						disabled: !app.migrated && !appCountQuery?.data?.hasUnlimitedApps && appCountQuery?.data?.enabled >= appCountQuery?.data?.limit,
-						action: handleEnable,
-					},
-				}),
+			...(isPossibleToEnableApp && {
```

Comment:
Would it be possible to refactor this object to be created outside this lines and avoid having a high cyclomatic complexity in here?

Reply:
Where do you think that could be a good place for this object?